pycaret/internal/plots/time_series.py

```diff
@@ -466,71 +488,20 @@ def plot_acf(
     elif time_series_name is not None:
         title = f"{title} | {time_series_name}"
 
-    lower_y = corr_array[1][:, 0] - corr_array[0]
-    upper_y = corr_array[1][:, 1] - corr_array[0]
-
-    fig = go.Figure()
-
-    fig.add_scattergl(
-        x=np.arange(len(corr_array[0])),
-        y=corr_array[0],
-        mode="markers",
-        marker_color="#1f77b4",
-        marker_size=10,
-        name="ACF",
-    )
-
-    [
-        fig.add_scattergl(
-            x=(x, x),
-            y=(0, corr_array[0][x]),
-            mode="lines",
-            line_color="#3f3f3f",
-            name=f"Lag{ind + 1}",
-        )
-        for ind, x in enumerate(range(len(corr_array[0])))
-    ]
-
-    # For some reason scattergl does not work here. Hence switching to scatter.
-    # (refer: https://github.com/pycaret/pycaret/issues/2211).
-    fig.add_scatter(
-        x=np.arange(len(corr_array[0])),
-        y=upper_y,
-        mode="lines",
-        line_color="rgba(255,255,255,0)",
-        name="UC",
-    )
-    fig.add_scatter(
-        x=np.arange(len(corr_array[0])),
-        y=lower_y,
-        mode="lines",
-        fillcolor="rgba(32, 146, 230,0.3)",
-        fill="tonexty",
-        line_color="rgba(255,255,255,0)",
-        name="LC",
-    )
-    fig.add_scattergl(
-        x=(0, len(corr_array[0])),
-        y=(0, 0),
-        mode="lines",
-        line_color="#3f3f3f",
-        name="",
-    )
-
     with fig.batch_update():
-        fig.update_xaxes(range=[-1, len(corr_array[0]) + 1])
-        fig.update_yaxes(zerolinecolor="#000000")
-
+        fig.update_xaxes(title_text="Lags", row=1, col=1)
+        fig.update_yaxes(title_text="ACF", row=1, col=1)
         template = _resolve_dict_keys(
             dict_=fig_kwargs, key="template", defaults=fig_defaults
         )
         fig.update_layout(title=title, showlegend=False, template=template)
+        fig.update_traces(marker={"size": 12})
```

Comment:
Can you change the size to 10 (not 12) in order to be consistent with the previous version? Thanks!